arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -0,0 +1,576 @@
+            "type": "string"
+          },
+          {
+            "name": "registryName",
+            "in": "path",
+            "description": "The container registry name.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        },
+        "deprecated": false
+      },
+      "patch": {
+        "tags": [
+          "Registries"
+        ],
+        "description": "Updates a container registry with the specified parameters. The storage account provided in this operation should be in the same location as the container registry.",
+        "operationId": "Registries_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The resource group name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "registryName",
+            "in": "path",
+            "description": "The container registry name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "registryUpdateParameters",
+            "in": "body",
+            "description": "The parameters for updating a container registry.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RegistryUpdateParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Registry"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerRegistry/registries": {
+      "get": {
+        "tags": [
+          "Registries"
+        ],
+        "description": "Lists all the container registries available under the given resource group.",
+        "operationId": "Registries_ListByResourceGroup",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The resource group name.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/RegistryListResult"
+            }
+          }
+        },
+        "deprecated": false,
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }

```

Comment:
@hovsepm - That would be inconsistent with our primary goal "Swagger spec should accurately describe what is there on the wire". Going from non-pageable to pageable is a breaking change. Wouldn't it be correct to consume a breaking change in the API because of that (whenever that happens) ?